### ISSUE TITLE
Navigation Screen Content `<nav>` element contains non-navigation content

### ISSUE THREAD

**Issue by jeryj** [WordPress/gutenberg#48652]
### Description

There are times in the navigation sidebar screen that have normal `<p>` content inside of a `<nav>` element, such as on the Site Editor -> Template -> Individual template.

The "Displays a single page." content is wrapped in a `<nav>` element, when it should be a more generic element or none at all.

### Step-by-step reproduction instructions

1. From /wp-admin
2. Appearance -> Editor in sidebar menu
3. Select "Templates" in the sidebar menu
4. Select "Page"
5. "Displays a single page." content is within a `<nav>` element

### Screenshots, screen recording, code snippet

<img width="1221" alt="Edit Page Template screen showing 'displays a single page' on the sidebar" src="https://user-images.githubusercontent.com/967608/222235570-7ad35ae5-043f-474b-9d59-12445518340b.png">

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes
